code/modules/vehicles/armored/armored_actions.dm

```diff
@@ -150,3 +150,25 @@
 	chassis.visible_message("[chassis] honks its horn!")
 	playsound(chassis, 'sound/vehicles/horns/armored_horn.ogg', 70)
 	TIMER_COOLDOWN_START(chassis, COOLDOWN_ARMORED_HORN, 15 SECONDS) //To keep people's eardrums intact
+
+/datum/action/vehicle/sealed/armored/strafe
+	name = "Toggle Strafing. Disabled when Alt is held."
+	action_icon_state = "strafe"
+	keybinding_signals = list(
+		KEYBINDING_NORMAL = COMSIG_MECHABILITY_TOGGLE_STRAFE,
+	)
+/datum/action/vehicle/sealed/armored/strafe/action_activate(trigger_flags)
+	if(!owner || !chassis || !(owner in chassis.occupants))
+		return
+	chassis.toggle_strafe()
+
+///Toggles strafemode on or off
+/obj/vehicle/sealed/armored/proc/toggle_strafe()
+	strafe = !strafe
+
+	to_chat(occupants, "strafing mode [strafe?"on":"off"].")
+	log_message("Toggled strafing mode [strafe?"on":"off"].", LOG_MECHA)
```

Comment:
why is this logged? why is it not a balloon alert?

Reply:
I copied your code from mech strafe noobgus